Gemfile

```diff
@@ -28,7 +28,7 @@ group :default do
 
   # Provides bulk insert capabilities
   gem 'activerecord-import'
-  gem 'record_loader'
+  gem 'record_loader', git: 'https://github.com/sanger/record_loader', branch: 'dpl680_upgrade_ruby3'
```

Comment:
What are you going to do with this. Does it need sorting once this is moved to production?

Reply:
No, I'll change it